ddtrace/_trace/tracer.py

```diff
@@ -291,7 +291,10 @@ def _atexit(self) -> None:
             self.SHUTDOWN_TIMEOUT,
             key,
         )
-        self.shutdown(timeout=self.SHUTDOWN_TIMEOUT)
+        try:
+            self.shutdown(timeout=self.SHUTDOWN_TIMEOUT)
+        except ValueError:  # hook cleanup can sometimes raise this
```

Comment:
The value error is likely raise here: https://github.com/DataDog/dd-trace-py/blob/9319ecf64d9cb0fb35a24339abec19151f942f06/ddtrace/internal/forksafe.py#L68. I think this behavior is intentional but I am not sure why we need to crash applications if we attempt to register a hook that doesn't exist.

Should we change this behavior for all ddtrace products?

Reply:
Seems worth a shot.